addons/website/i18n/website.pot

```diff
@@ -2892,7 +2892,7 @@ msgstr ""
 #: code:addons/website/static/src/js/tours/tour_utils.js:0
 #, python-format
 msgid ""
-"Drag the <b>${snippet.name}</b> building block and drop it in your page."
+"Drag the <b>%s</b> building block and drop it in the bottom of your page."
```

Comment:
Rebase your branch, @JKE-be made a commit changing this sentence last week (without fixing translation like you do here) so there will be a conflict here :wink: But I think JKE use the correct english "at the bottom" and not "in the bottom"... right?